bool targetInSight
float32 delX
float32 delY
float32 delZ
float32 outX
float32 outY
float32 outZ
float32 outPsi
float32 fwdTgtdot
float32 fwdTgtddot
